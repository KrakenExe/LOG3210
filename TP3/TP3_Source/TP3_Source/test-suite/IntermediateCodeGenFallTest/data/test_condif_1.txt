Declaration
num a;
EndDeclaration

if(false) {
  a = 42;
}
else {
  a = 3210;
}

