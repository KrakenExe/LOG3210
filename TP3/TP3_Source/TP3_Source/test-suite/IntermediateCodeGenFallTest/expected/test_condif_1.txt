goto _L1
a = 42
goto _L0
_L1
a = 3210
_L0
